beacon_node/http_api/src/lib.rs

```diff
@@ -1933,6 +1933,51 @@ pub fn serve<T: BeaconChainTypes>(
             })
         });
 
+    /*
+     * builder/states
+     */
+
+    let builder_states_path = eth_v1
+        .and(warp::path("builder"))
+        .and(warp::path("states"))
+        .and(chain_filter.clone());
+
+    // GET builder/states/{state_id}/expected_withdrawals
+    let get_expected_withdrawals = builder_states_path
+        .clone()
+        .and(warp::path::param::<StateId>())
+        .and(warp::path("expected_withdrawals"))
+        .and(warp::path::end())
+        .and_then(|chain: Arc<BeaconChain<T>>, state_id: StateId| {
+            blocking_json_task(move || {
```

Comment:
Looks like this endpoint also need to support SSZ responses:
https://github.com/ethereum/beacon-APIs/blob/a62aa810fb283873de0d8025405f7a8ce6991f51/apis/builder/states/expected_withdrawals.yaml#L37-L39